wren-launcher/commands/launch.go

```
@@ -375,6 +375,7 @@ func validateOpenaiApiKey(apiKey string) bool {
 	// insufficient credit balance error
 	if err != nil {
 		pterm.Error.Println("Invalid API key", err)
+		fmt.Scanln()
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)